RFC/src/SUMMARY.md

```diff
@@ -1,39 +1,41 @@
 # Tari RFC documents
 
 [About the Tari RFC documents](about.md)
+
 - [RFC-0001: An overview of the Tari network](RFC-0001_overview.md)
-- [RFC-0010: Tari code structure and organisation](RFC-0010_CodeStructure.md)
+- [RFC-0010: Tari code structure and organization](RFC-0010_CodeStructure.md)
 - [RFC-0100: The Tari Base Layer](RFC-0100_BaseLayer.md)
-    - [RFC-0110: Base nodes](RFC-0110_BaseNodes.md)
-  -   [RFC-0111: Base node architecture](RFC-0111_BaseNodeArchitecture.md)
-    - [RFC-0120: Consensus rules](RFC-0120_Consensus.md)
-    - [RFC-0131: Mining](RFC-0131_Mining.md)
-    - [RFC-0140: Sync and Seeding](RFC-0140_Syncing_and_seeding.md)
-    - [RFC-0150: Wallets](RFC-0150_Wallets.md)
-    - [RFC-0151: Transaction protocol](RFC-0151_TransactionProtocol.md)
-    - [RFC-0152: Emoji ID](RFC-0152_EmojiId.md)
-    - [RFC-0170: Network Communication Protocol](RFC-0170_NetworkCommunicationProtocol.md)
-      - [RFC-0171: Message Serialisation](RFC-0171_MessageSerialisation.md)
-      - [RFC-0172: Peer to Peer Messaging Protocol](RFC-0172_PeerToPeerMessagingProtocol.md)
-    - [RFC-0190: Mempool](RFC-0190_Mempool.md)
-    - [Tari-specific extensions to Mimblewimble](BaseLayerExtensions.md)
-      - [RFC-0201: TariScript](RFC-0201_TariScript.md)
-      - [RFC-0202: TariScript Opcodes](RFC-0202_TariScriptOpcodes.md)
-      - [RFC-0230: Hash time locked contracts](RFC-0230_HTLC.md)
-      - [RFC-0322: Validator Node Registration](RFC-0322_VNRegistration.md)
-      - [RFC-0341: Asset registration](RFC-0341_AssetRegistration.md)
+  - [RFC-0110: Base nodes](RFC-0110_BaseNodes.md)
+  - [RFC-0111: Base node architecture](RFC-0111_BaseNodeArchitecture.md)
+  - [RFC-0120: Consensus rules](RFC-0120_Consensus.md)
+  - [RFC-0131: Mining](RFC-0131_Mining.md)
+  - [RFC-0140: Sync and Seeding](RFC-0140_Syncing_and_seeding.md)
+  - [RFC-0150: Wallets](RFC-0150_Wallets.md)
+  - [RFC-0151: Transaction protocol](RFC-0151_TransactionProtocol.md)
+  - [RFC-0152: Emoji ID](RFC-0152_EmojiId.md)
+  - [RFC-0170: Network Communication Protocol](RFC-0170_NetworkCommunicationProtocol.md)
+    - [RFC-0171: Message Serialisation](RFC-0171_MessageSerialisation.md)
+    - [RFC-0172: Peer to Peer Messaging Protocol](RFC-0172_PeerToPeerMessagingProtocol.md)
+  - [RFC-0190: Mempool](RFC-0190_Mempool.md)
+  - [Tari-specific extensions to Mimblewimble](BaseLayerExtensions.md)
```

Comment:
Not sure what the correct indentation should here? maybe this?
```suggestion
- [Tari-specific extensions to Mimblewimble](BaseLayerExtensions.md)
```

Reply:
from [rfc.tari.com](//rfc.tari.com) it looks like it should be in line with mempool (and heads up the 5 extensions) so I think this is right?

<img width="253" alt="image" src="https://user-images.githubusercontent.com/351403/121415202-c2af1280-c967-11eb-8ed4-595fb13133c0.png">